### PR TITLE
feat: add the machines package

### PR DESCRIPTION
**Pull request by renan061**

### CODE REVIEW

**Review by torives — DISMISSED** [review]

**Review by torives — APPROVED** [review]

File: internal/node/config/generate/Config.toml [hunk 1]
```diff
@@ -250,3 +256,13 @@ go-type = "bool"
...
+
+#
+# Temporary
+#
+
+[temp.CARTESI_MACHINE_SERVER_VERBOSITY]
+default = "info"
+go-type = "string"
+description = """
+TODO."""
```
Comment (marcelstanley) [inline internal/node/config/generate/Config.toml:268]:
```suggestion
```

File: docs/config.md [hunk 2]
```diff
@@ -258,3 +265,10 @@ How many seconds the node will wait before trying to finish epochs for all appli
 Path to the directory with the cartesi-machine snapshot that will be loaded by the node.
 
 * **Type:** `string`
+
+## `CARTESI_MACHINE_SERVER_VERBOSITY`
+
+TODO.
```
[the remark this answers is not in the document]
Comment (marcelstanley) [reply docs/config.md:271]: ☝️

File: internal/node/config/generate/Config.toml (hunk 1 above)
Comment (vfusco) [reply internal/node/config/generate/Config.toml:268]: will be fixed later

File: docs/config.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (vfusco) [reply docs/config.md:271]: will be fixed later

**Pull request merged**